src/components/Login/EmailLogin.tsx

```diff
@@ -69,7 +94,7 @@ export default function EmailLogin(props: Props) {
     setIsValidEmail(validation);
     if (validation) {
       inputEmail(input);
-      handleEmailCheck();
+      handleEmailCheck().then();
```

Comment:
Adding .then() is a standard way to handle the "promise" returned from a method :) 
Nothing in the brackets means we won't use the "promise" returned

Reply:
I think this _then()_ is not needed, but I don't know how to explain it.
@tatoMa Do you think this _then()_ is needed?

Reply:
I added this because my WebStorm always reminds me about adding .then() since it belongs to code smells.

However, I think the reminder may be too rigid since there are some situations in that we've already resolved everything we should, like above :)